src/App/Identity/identity/Accessibility/index.js

```diff
@@ -4,10 +4,13 @@ import * as externalLinks from "./constants";
 import { DocContainer } from "@docutils";
 
 const basename = process.env.basename;
+const brandTitle = process.env.brandTitle;
+
+const otherLinks = brandTitle === "Swedbank Pay" ? externalLinks.othersAboutAccessibility : externalLinks.othersAboutAccessibility.slice(1);
```

Comment:
I am skeptical about this solution but I guess it works as a quick fix for now ;)

Reply:
I know this is not scaleable at all. Will be fixed once we get some real content :)